src/pages/Login/Login.scss

```
@@ -0,0 +1,71 @@
+.signIn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  height: 100vh;
+  margin: 0 60px;
+  .loginTitle {
+    padding-bottom: 40px;
+    font-size: 32px;
+    font-weight: 700;
+  }
+  .loginInputContainer {
```

```suggestion
  margin: 0 60px;
  
  .loginTitle {
    padding-bottom: 40px;
    font-size: 32px;
    font-weight: 700;
  }
  
  .loginInputContainer {
```

스타일 블록끼리 가독성을 위해 한칸씩 띄워주세요~